data/json/requirements/toolsets.json

```diff
@@ -40,7 +40,7 @@
     "type": "requirement",
     "//": "Includes forging resources as well as tools needed for most blacksmithing",
     "qualities": [ { "id": "ANVIL", "level": 3 }, { "id": "HAMMER", "level": 3 } ],
-    "tools": [ [ [ "forge", 20 ], [ "oxy_torch", 20 ], [ "fake_arc_furnace", 10 ] ], [ [ "tongs", -1 ] ] ]
+    "tools": [ [ [ "forge", 20 ], [ "oxy_torch", 20 ], [ "fake_arc_furnace", 10 ] ], [ [ "metal_file", -1 ], [ "tongs", -1 ] ] ]
```

Comment:
```suggestion
    "tools": [ [ [ "forge", 20 ], [ "oxy_torch", 20 ], [ "fake_arc_furnace", 10 ] ], [ [ "metal_file", -1 ] ], [ [ "tongs", -1 ] ] ]
```

Reply:
Commit this suggestion if you are wanting both file and tongs to be required as apposed to one or the other as it is currently written in your PR.   Also you could replace the metal file in tools with a Filing 2 quality near hammer and anvil.

Reply:
The filing quality would allow things like a set of metal files to work as long as they have the quality while the tool requires the specific tool and only that tool

Reply:
[ [ "grind", -2 ] ], [ [ "tongs", -1 ] ] ] Would this be how I set it up to use the GRIND quality at level 2 or do I need to do something else?